config/unix-cuda.cmake

```diff
@@ -45,7 +45,8 @@ endif()
 if( NOT CUDA_FLAGS_INITIALIZED )
 
   set( CUDA_FLAGS_INITIALIZED "yes" CACHE INTERNAL "using draco settings." )
-  set( CMAKE_CUDA_FLAGS                "${Draco_CUDA_ARCH} -g --expt-relaxed-constexpr")
+  set( CMAKE_CUDA_FLAGS                "${Draco_CUDA_ARCH} -g
+    --expt-relaxed-constexpr --expt-extended-lambda")
```

Comment:
The line wrap will break cmake.  Can you change this to 

```cmake
string(APPEND CMAKE_CUDA_FLAGS "-g --expt-relaxed-constexpr --expt-extended-lambda")
```

Do we still want `-g` for all cuda compiles?  Should `-g` be moved to `DEBUG` flags?